src/index.ts

```diff
@@ -296,6 +263,21 @@ export default <ExportedHandler<Env>>{
 										},
 									],
 								},
+								{
+									type: ComponentType.ActionRow,
+									components: [
+										{
+											type: ComponentType.TextInput,
+											style: TextInputStyle.Paragraph,
+											custom_id: "thread-name-input",
+											label: "Thread name",
+											placeholder: "What should I name your discussion thread?",
+											min_length: 5,
+											max_length: 20,
+											required: false,
+										},
+									],
+								}
```

Comment:
This is also a not-necessary nit, but it would make a little more sense to put the title first